lib/logstash_sidekiq_logger/formatter.rb

```diff
@@ -0,0 +1,111 @@
+module LogstashSidekiqLogger
+  class Formatter < Logger::Formatter
+    def call(_severity, _time, _program_name, message)
+      store_message_failure_metadata(message)
+
+      performed_job = RequestStore.store[:performed_job]
+      return unless performed_job && message_needs_logging?(message)
+
+      data = extract_log_data(performed_job, message)
+      event = LogStash::Event.new(data)
+
+      RequestStore.clear!
+
+      # Is there a better way to make the logs appear on new lines??
```

Comment:
This comment feels kinda pointless (unless it's a comment to reviewers); either remove it, or find the solution?